modules/dspm-environments/main.tf

```diff
@@ -1,6 +1,10 @@
 data "aws_region" "current" {}
```

Comment:
The `local.availability_zones` variable needs to be removed since it is no longer used